### ISSUE TITLE
Translation and scaling behave strangely somewhere

### ISSUE THREAD

**Issue by comodoro** [comodoro/FractalZoo#15]
Not corresponding to touches

**Comment by TermanEmil** [comment]
It seems that the Translation depends on the distance from the center of the screen.

If you try to translate it from the corner of the screen, the image will travel much further

**Comment by comodoro** [comment]
You are right! Now to find out what is the reason, because I think some fractals behaved more or less correctly. It may be the view that displays the shaders or the shaders themselves.
